src/components/shared/dynamicFee/dynamicFee.css

```diff
@@ -0,0 +1,85 @@
+@import '../../../app/mixins.css';
+
+.fieldGroup {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 16px;
+  padding: 0 var(--horizontal-padding-l);
+
+  &.dynamicFeeWrapper {
+    flex-direction: row;
+    margin-bottom: 42px;
+  }
+}
+
+.fieldLabel {
+  @mixin contentNormal bold;
+
+  align-items: center;
+  color: var(--color-maastricht-blue);
+  display: flex;
+  margin-bottom: 8px;
+
+  & .loading {
+    color: var(--color-ultramarine-blue);
+    margin-left: 5px;
+  }
+}
+
+.dynamicFeeWrapper {
+  & > .col:first-child {
+    padding-right: 100px;
+  }
+
+  & > .col {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 50%;
+    justify-content: space-between;
+    height: 74px;
+
+    & > .fieldLabel {
+      margin-bottom: 14px;
+    }
+
+    & > .prioritySelector {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      margin-bottom: 4px;
+
+      & > .feePriority {
```

Comment:
Not sure if this name makes sense. Fee and priority are different identities. I think it's meant to be `.priorityTitle`